tests/Issues/IfElseInline/config/configured_rule.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\CodeQuality\Rector\If_\SimplifyIfElseToTernaryRector;
+use Rector\CodingStyle\Rector\Property\InlineSimplePropertyAnnotationRector;
+use Rector\Config\RectorConfig;
+use Rector\DeadCode\Rector\Assign\RemoveUnusedVariableAssignRector;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->importNames();
```

Comment:
the issue is happen only on auto import enabled, so the config test require auto import to be enabled to reproduce it.